sql/parser/builtins.go

```diff
@@ -761,3 +774,12 @@ func datumToRawString(datum Datum) (string, error) {
 
 	return "", fmt.Errorf("argument type unsupported: %s", datum.Type())
 }
+
+func round(x float64, n int64) (Datum, error) {
+	if n < 0 || n > 50 {
```

Comment:
Will set the boundary to 325 and open a follow-up PR. Thanks for pointing out the issue.

I think, a reasonable boundary is the sum of exponent and significant digits 308 + 15 = 323. This matches the comment in [round](https://hg.python.org/cpython/file/v3.5.0/Objects/floatobject.c#l863) of python.

See example from the python shell below
```
round(1.1e-308, 308) == 1.1e-308  # false
round(1.1e-308, 309) == 1.1e-308  # true
# ...
round(0.000000000000001e-308, 308) == 0.000000000000001e-308  # false
round(0.000000000000001e-308, 325) == 0.000000000000001e-308  # true
```